e2e/tests/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -0,0 +1,101 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { test, expect } = require('@playwright/test');
+
+test.describe('Telemetry Table', () => {
+    test('unpauses when paused by button and user changes bounds', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5113'
+        });
+
+        const bannerMessage = '.c-message-banner__message';
+        const bannerCloseButton = '.c-message-banner__close-button';
+        const createButton = 'button:has-text("Create")';
+
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Click create button
+        await page.locator(createButton).click();
+        await page.locator('li:has-text("Telemetry Table")').click();
+        await Promise.all([
+            page.waitForNavigation(),
+            page.locator('text=OK').click(),
+            // Wait for Save Banner to appear
+            page.waitForSelector(bannerMessage)
+        ]);
+
+        // Wait until Save Banner is gone
+        await page.locator(bannerCloseButton).click();
+        await page.waitForSelector(bannerMessage, { state: 'detached'});
+
+        // Save (exit edit mode)
+        await page.locator('text=Snapshot Save and Finish Editing Save and Continue Editing >> button').nth(3).click();
```

Comment:
This is what I was referring to in Teams. We'll need to spend a bit of time updating the application with better locator hooks